### PR TITLE
Windows-friendly handler file names 

### PR DESCRIPTION
**Pull request by spcfran**
as per #1 

Also renamed `route` to `routeFileName` to make it clear it does not refer to the routes in express, and added some doc

### CODE REVIEW

File: deconstruct.js [hunk 1]
```diff
@@ -198,7 +198,18 @@ app.use ( ( req, res, next ) => {
 module.exports = {
     setSecretRetriever: secretRetriever => utils.auth.getIssuerSecret = secretRetriever,
     addUtil: ( name, util ) => { utils[name] = util; },
-    loadRoutes: ( routeDir, callback ) => {
+    loadRoutes: ( options, callback ) => {
+        const routeDir = options;
+        const pathParameterFlag = ':';
+
+        if (typeof options === 'object') {
+            routeDir = options.routeDir;
```
Comment (adriaan-pelzer) [inline deconstruct.js]: Would you handle the case where options.routeDir is undefined? Throwing is ok for now, I think.

Comment (adriaan-pelzer) [reply deconstruct.js]: Another thing: Will this actually work? Will JS allow you to change the value of a const?

Comment (adriaan-pelzer) [reply deconstruct.js]: how about:
const routeDir = (typeof options === 'string') ? (options || 'routes') : (options.routeDir || 'routes');

**Review by adriaan-pelzer — APPROVED** [review]

File: deconstruct.js (hunk 1 above)
Comment (spcfran) [reply deconstruct.js]: > Another thing: Will this actually work? Will JS allow you to change the value of a const?

I got so used to rely on eslint/typescript in other projects that I didn't even notice until that errored at runtime :)

**Comment by spcfran** [comment]
Closes #1

**Pull request merged**